hal/stm32f373/i2c.h

```diff
@@ -13,7 +13,7 @@
 #define __I2C__
 
 #define I2C_TIMEOUT         ((uint32_t)0x10)
-#define I2C_LONG_TIMEOUT         ((uint32_t)(10 * I2C_FLAG_TIMEOUT)) 
+#define I2C_LONG_TIMEOUT    ((uint32_t)(I2C_FLAG_TIMEOUT)) 
```

Comment:
I can see this is not your code ... but I don't get any of this, so this is the mapping:

```
#define  I2C_FLAG_TIMEOUT               I2C_ISR_TIMEOUT
#define  I2C_ISR_TIMEOUT                     ((uint32_t)0x00001000)        /*!< Timeout or Tlow detection flag */
```
which maps to the I2C_TIMEOUT register TIDLE bit12 or I2Cx_ISR TIMEOUT bit12
![image](https://user-images.githubusercontent.com/335300/47754604-50321c80-dd00-11e8-9df2-2fde21ceb360.png)

But we use it as a counter but it is just a bitmask ?